sklearn/random_projection.py

```diff
@@ -82,10 +82,12 @@ def johnson_lindenstrauss_min_dim(n_samples, *, eps=0.1):
         Number of samples. If an array is given, it will compute
         a safe number of components array-wise.
 
-    eps : float or numpy array of float in ]0,1[, optional (default=0.1)
-        Maximum distortion rate as defined by the Johnson-Lindenstrauss lemma.
+    eps : float or ndarray of shape (n_components,), dtype=float, default=0.1
+        float in range(0, 1)
+        Maximum distortion rate as defined by the \
```

Comment:
You don't need to use `\`

Please articulate the documentation such that it will be easy to read

```python
Maximum distortion rate in the range `(0, 1)` as defined by the
Johnson-Lindenstrauss lemma.
```